lib/jazzy/assets/css/jazzy.css.scss

```diff
@@ -406,258 +361,29 @@ header {
   }
 }
 
-// Asides
-
-div.Swift {
-  padding: 4px 0 2px 10px;
-  margin-bottom: 1em;
-  border-left: $declaration_swift_border;
-  overflow-y: hidden;
-  .aside-title {
-    color: $declaration_lang_title_swift_color;
-    -webkit-user-select: none;
-  }
-}
-
-div.attention {
-  padding: 4px 0 2px 10px;
-  margin-bottom: 1em;
-  border-left: $aside_attention_border;
-  overflow-y: hidden;
-  .aside-title {
-    color: $aside_attention_color;
-    -webkit-user-select: none;
-  }
-}
-
-div.author {
-  padding: 4px 0 2px 10px;
-  margin-bottom: 1em;
-  border-left: $aside_author_border;
-  overflow-y: hidden;
-  .aside-title {
-    color: $aside_author_color;
-    -webkit-user-select: none;
-  }
-}
-
-div.authors {
-  padding: 4px 0 2px 10px;
-  margin-bottom: 1em;
-  border-left: $aside_authors_border;
-  overflow-y: hidden;
-  .aside-title {
-    color: $aside_authors_color;
-    -webkit-user-select: none;
-  }
-}
-
-div.bug {
-  padding: 4px 0 2px 10px;
-  margin-bottom: 1em;
-  border-left: $aside_bug_border;
-  overflow-y: hidden;
-  .aside-title {
-    color: $aside_bug_color;
-    -webkit-user-select: none;
-  }
-}
-
-div.complexity {
-  padding: 4px 0 2px 10px;
-  margin-bottom: 1em;
-  border-left: $aside_complexity_border;
-  overflow-y: hidden;
-  .aside-title {
-    color: $aside_complexity_color;
-    -webkit-user-select: none;
-  }
-}
-
-div.copyright {
-  padding: 4px 0 2px 10px;
-  margin-bottom: 1em;
-  border-left: $aside_copyright_border;
-  overflow-y: hidden;
-  .aside-title {
-    color: $aside_copyright_color;
-    -webkit-user-select: none;
-  }
-}
-
-div.date {
-  padding: 4px 0 2px 10px;
-  margin-bottom: 1em;
-  border-left: $aside_date_border;
-  overflow-y: hidden;
-  .aside-title {
-    color: $aside_date_color;
-    -webkit-user-select: none;
-  }
-}
-
-div.experiment {
-  padding: 4px 0 2px 10px;
-  margin-bottom: 1em;
-  border-left: $aside_experiment_border;
-  overflow-y: hidden;
-  .aside-title {
-    color: $aside_experiment_color;
-    -webkit-user-select: none;
-  }
-}
-
-div.important {
-  padding: 4px 0 2px 10px;
-  margin-bottom: 1em;
-  border-left: $aside_important_border;
-  overflow-y: hidden;
-  .aside-title {
-    color: $aside_important_color;
-    -webkit-user-select: none;
-  }
-}
-
-div.invariant {
-  padding: 4px 0 2px 10px;
-  margin-bottom: 1em;
-  border-left: $aside_invariant_border;
-  overflow-y: hidden;
-  .aside-title {
-    color: $aside_invariant_color;
-    -webkit-user-select: none;
-  }
-}
-
-div.note {
-  padding: 4px 0 2px 10px;
-  margin-bottom: 1em;
-  border-left: $aside_note_border;
+.aside, .Swift {
+  padding: 0.4em 0.8em;
```

Comment:
why the change from px to em? I prefer the spacing from before.

**before**
![image](https://cloud.githubusercontent.com/assets/474794/10805865/597a026e-7d8e-11e5-8f30-e64a4d2e0974.png)

**after**
![image](https://cloud.githubusercontent.com/assets/474794/10805871/677c5d44-7d8e-11e5-8525-8caaea6ff98b.png)